README.md

```diff
@@ -53,6 +54,44 @@ Install with your preferred package manager:
 }
 ```
 
+packer.nvim
+
+```lua
+use {
+    'smoka7/multicursors.nvim',
+    requires = {
+        'nvim-treesitter/nvim-treesitter',
+        'smoka7/hydra.nvim',
+    },
+    config = function()
+        local N = require 'multicursors.normal_mode'
+        local I = require 'multicursors.insert_mode'
+        requires('multicursors').setup {
+            normal_keys = {
+                -- to change default lhs of key mapping change the key
+                [','] = {
+                    -- assigning nil to method exits from multi cursor mode
+                    method = N.clear_others,
+                    -- you can pass :map-arguments here
+                    opts = { desc = 'Clear others' },
+                },
+            },
+            insert_keys = {
+                -- to change default lhs of key mapping change the key
+                ['<CR>'] = {
+                    -- assigning nil to method exits from multi cursor mode
+                    method = I.Cr_method,
+                    -- you can pass :map-arguments here
+                    opts = { desc = 'New line' },
+                },
+            },
+        }
+
+        keymap({ 'v', 'n' }, '<Leader>m', requires('multicursors').start, { desc = 'Create a selection for selcted text or word under the cursor' })
+    end,
+}
+```
```

Comment:
```suggestion
```

Reply:
I think adding explanation for how to set it up with packer going to be a good idea. Also I still think that just the plugin and it's requires is enough for all plugin managers like so:
```lua
-- packer.nvim
use {
    'smoka7/multicursors.nvim',
    requires = {
        'nvim-treesitter/nvim-treesitter',
        'smoka7/hydra.nvim',
    }
}```

```lua
-- lazy.nvim
{
    "smoka7/multicursors.nvim",
    event = "VeryLazy",
    dependencies = {
        'nvim-treesitter/nvim-treesitter',
        'smoka7/hydra.nvim',
    },
}```

the other stuff we can move to another section for setup I think.

Reply:
Ok, I'll have to add some documents about how mappings work. I move it then.
I think your work with this pr is done. Thank you.

Reply:
No problem buddy.

Always open to help, this sounds like a great plugin I'd love to help with it.